Fixes path to a2ps binary in a2ps.el

See: https://patches.ubuntu.com/by-release/extracted/ubuntu/a/a2ps/1:4.13c~rc5-1/01_a2ps_el_path_fix.dpatch

## 01_a2ps_el_path_fix.dpatch by Masayuki Hatta <mhatta@debian.org>

Index: b/contrib/emacs/a2ps.el
===================================================================
--- a/contrib/emacs/a2ps.el
+++ b/contrib/emacs/a2ps.el
@@ -44,7 +44,7 @@
 ;; a2ps-compile-regexp
 
 ;;path to the a2ps program
-(defvar a2ps-program "/usr/local/bin/a2ps")
+(defvar a2ps-program "/usr/bin/a2ps")
 
 ;;thank god for make-regexp.el!
 (defvar a2ps-font-lock-keywords
